components/admin_console/workspace-optimization/overall-score.tsx

```diff
@@ -4,19 +4,22 @@ import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
 import CircularChart from 'components/common/circular_chart/circular_chart';
+import WarningSvg from 'components/common/svg_images_components/warning_svg';
```

Comment:
SH00T ... I have forgotten to tell you that in compass-icons I built SVG components for the icons as well in version 0.1.19 ... so if we upgrade the version you can just use those and remove the svgs from the common folder .... i am sorry i haven't told you earlier! If you have questions regarding this feel free to ping me!

Reply:
no worries, I already created a card in the actions board to replace all the SVGs for the compass icons in a different pull request so we can move forward with this one and avoid another big merge conflict with the base branch.

https://community.mattermost.com/boards/workspace/7dqusmf8stn3dens5ckhyatbdo/boattb15z9fr4m85etwrdekkduh/v37ny5c5eafnk3dypexwzagip7a/c74odxpk9ppr18y6wt8prnt3boh